go.mod

```diff
@@ -24,17 +26,20 @@ require (
 	github.com/onsi/gomega v1.13.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cast v1.4.0 // indirect
-	github.com/spf13/cobra v1.2.1
```

Comment:
Do we need this downgrade?

Reply:
Probably not, probably it has been downgraded when i was chacking gcloud sdk versions. I have to check, thanks